### ISSUE TITLE
Combining language and dates throws an error

### ISSUE THREAD

**Issue by czpython** [KristianOellegaard/django-hvad#319]
Hello,

Given the following model:

``` python
class I18nModel(TranslatableModel):
    translations = TranslatedFields(
        title=models.CharField(_('title'), max_length=100),
    )
    date = models.DateField(_('date'), null=False)
```

A query using language and dates fails:

``` python
I18nModel.objects.language('en').dates('date', 'year')
```

With the following error:

```
Traceback:
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  132.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/cms/utils/decorators.py" in inner
  18.         return func(request, *args, **kwargs)
File "/home/hvad-playground/app/src/pom/views.py" in archive
  216.             '-date').dates('date', 'year'),
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/hvad/manager.py" in dates
  744.         return super(TranslationQueryset, self).dates(field_name, kind=kind, order=order)
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/django/db/models/query.py" in dates
  631.             'datefield', flat=True
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/hvad/manager.py" in values_list
  739.         return super(TranslationQueryset, self).values_list(*fields, **kwargs)
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/django/db/models/query.py" in values_list
  616.                 _fields=fields)
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/hvad/manager.py" in _clone
  244.         return super(TranslationQueryset, self)._clone(**kwargs)
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/django/db/models/query.py" in _clone
  960.             c._setup_query()
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/django/db/models/query.py" in _setup_query
  1136.         self.query.add_fields(self.field_names, True)
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/django/db/models/sql/query.py" in add_fields
  1699.                 targets, final_alias, joins = self.trim_joins(targets, joins, path)
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/django/db/models/sql/query.py" in trim_joins
  1510.             cur_targets = set(t.column for t in targets)
File "/home/hvad-playground/app/local/lib/python2.7/site-packages/django/db/models/sql/query.py" in <genexpr>
  1510.             cur_targets = set(t.column for t in targets)

Exception Type: AttributeError at /en/hvad-test/
Exception Value: 'Date' object has no attribute 'column'
```

Environment:

Django Version: 1.8.15
Python Version: 2.7.12
Django Hvad: 1.6.0

Looks like a regression as it was working in previous versions.

**Comment by spectras** [comment]
Hello, thanks for reporting.
It looks related to #318.
Django changed the way it handles "special" querysets (date, values, values_list), which used to be subclasses and are now some sort of converter objects. Hvad has to be able to deal with both as long as it supports version on both side of the changing time point.

**Comment by spectras** [comment]
Oddly enough, I copied your model into a new project and ran the query and it seems to be working fine.
What database backend are you using?

**Comment by spectras** [comment]
Fixed on master. Fix will be included in v1.7.0.
